.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,19 +4,19 @@ about: Create a report to help us improve
 
 ---
 
-**Expected behavior**
+## Expected behavior ##
 
 A clear and concise description of what you want to do and what you think should happen. (Code to reproduce the behavior can be added below).
```

Comment:
@orbeckst What about putting the explanations into HTML comments so that they won't appear in the "published" issue?

Reply:
Good idea.